`<select>` element UI changes in 142 (#3658)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/release-notes/142.md

* [if()](https://developer.mozilla.org/docs/Web/CSS/if) at MDN.

<!-- ====================================================================== -->
## UI features

Below are user interface changes in Microsoft Edge.

<!-- ------------------------------ -->
#### Mobile and desktop parity for `<select>` elements

`<select>` HTML elements, when using the `size` and `multiple` attributes, now appear the same way on desktop and mobile devices.

* The `<select>` element now always appears as a list when the `size` attribute value is greater than `1`, both on desktop and mobile devices:

  ![A \<select\> element showing all options at once in the page.](./142-images/select-list.png)

>   Previously, mobile devices ignored the `size` attribute and always rendered the `<select>` element as a button that's used to display the options in a popup.

Reviewer: ```suggestion
  Previously, mobile devices ignored the `size` attribute and always rendered the `<select>` element as a dropdown list.
```
corp style guide:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dropdown
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/pop-up - "Don't use pop-up as noun"

* The `<select>` element now also always appears as a list when the `multiple` attribute is set, even if the `size` attribute is missing, both on desktop and mobile devices.

>   Previously, mobile devices rendered the `<select multiple>` element as a button that's used to display the options in a popup if the `size` attribute was missing.

Reviewer: done
